Translate Cleric (10085) (#474)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10085.txt

default
クレリック,Cleric
EU85_handJobAnime,

> 　癒しの力を使う聖職者。防御を固めているときは盾の女神に祈りをささげており、その力でマスターに山札のカードをもたらしている。 　争いよりも癒しと守りを重んじており、対戦相手であっても傷つけたくないと思っている。昨今の戦いでは性的攻撃によって男性を無力化できるため、杖で相手を叩くことに罪悪感を感じていた彼女は内心喜んでいる。,A cleric who uses healing magic. To defend herself; she'll pray to the Holy Shield Maiden and transfer cards from the deck to her master's hand. She values healing and protection over conflict; and prefers not to harm even her opponents. Lately; she's found a way to incapacitate men through sexual attacks; so she secretly feel pleased about being able to strike her opponents guilt-free with her staff. 

Reviewer: All things considered, this blurb isn't too bad I think! Just some suggestions:

The last sentence translation is not quite right. She's relieved that she doesn't need to strike opponent's with her staff anymore. NOT that she's happy she can.
```suggestion
　癒しの力を使う聖職者。防御を固めているときは盾の女神に祈りをささげており、その力でマスターに山札のカードをもたらしている。 　争いよりも癒しと守りを重んじており、対戦相手であっても傷つけたくないと思っている。昨今の戦いでは性的攻撃によって男性を無力化できるため、杖で相手を叩くことに罪悪感を感じていた彼女は内心喜んでいる。,A cleric who uses healing magic. When defending herself; she'll pray to the Shield Goddess and use that power to transfer cards from the deck to her master's hand. She values healing and protection over conflict; and prefers not to harm even her opponents. In the last few battles; she discovered she was able to deal with men through sexual attacks instead of striking opponent's with her staff; which she is secretly pleased about.
```

\n　また男性を甘やかすことに庇護欲が満たされる思いを感じており、相手がリラックスし自分に身をゆだねてくれるのを見るのが何よりも好きである。,\n She even feels a sense of fulfillment in pampering men; and derives great pleasure in seeing her targets relax and entrust themselves to her.

> [1].１ターンに１度、自分フィールド上のモンスター１体を選択して発動できる。そのモンスターのHPを500ポイント回復し、最大HPを300ポイントアップする。,[1]. Once per turn; you can activate this effect by selecting one monster on your field. Restore 500 HP to that monster and increase its Max HP by 300.

Reviewer: Nothing wrong with this translation! But we can localize it to be shorter:

```suggestion
[1].１ターンに１度、自分フィールド上のモンスター１体を選択して発動できる。そのモンスターのHPを500ポイント回復し、最大HPを300ポイントアップする。,[1]. Once per turn; you may select one monster on your field. Restore 500 HP to that monster and increase its Max HP by 300.
```

> [2].このカードが守備状態の場合、自分ターン終了時に発動する。デッキからカードを１枚ドローする。,[2].If this card is in defense position; activate this effect at the end of your turn. Draw one card from your deck.

Reviewer: ```suggestion
[2].このカードが守備状態の場合、自分ターン終了時に発動する。デッキからカードを１枚ドローする。,[2]. If this card is in defense position; you may draw one card from your deck and the end of your turn.
```

> [3].自分フィールド上のモンスターが１度に1500ポイント以上の戦闘ダメージ・効果ダメージを受けたときに発動する。そのモンスターのHPを1000ポイント回復する。,[3].Activate this effect when a monster on your field takes one instance of 1500 or greater battle damage or effect damage. Restore that monster's HP by 1000 points.

Reviewer: ```suggestion
[3].自分フィールド上のモンスターが１度に1500ポイント以上の戦闘ダメージ・効果ダメージを受けたときに発動する。そのモンスターのHPを1000ポイント回復する。,[3]. When a monster on your field recieves battle damage or effect damage of 1500 points or greater; restore that monster's HP by 1000 points.
```

> あなたに癒しを…。\nといっても敵を回復させるわけには行きません。\n心の癒しで我慢してくださいね❤,I only wish to heal you... Yet I shouldn't be healing an opponent. Please be content with this emotional healing instead; okay?❤

Reviewer: Yet I shouldn't be healing an opponent. "わけには行きません" is quite formal and is more like "should not/do not plan on"
```suggestion
あなたに癒しを…。\nといっても敵を回復させるわけには行きません。\n心の癒しで我慢してくださいね❤,I only wish to heal you... But I have no restore the health of my enemies. Allow me to heal your heart instead; okay?❤
```

心地いいですか？\n胸は大きくありませんが、愛情は人一倍ありますから、\n力を抜いてわたしに甘えてください…❤,Does it feel good? Though my chest may be modest; I'm certain I have more love to give than anyone else. Please; just relax and let me pamper you...❤
男の方のたくましい筋肉にわたしはかないません。\nこの細腕でできるのは、せいぜいあなたに慈悲を乞うことだけ。,I can't compete with a man's muscles. All I can use my thin arms for is to beg for mercy. 
例えば、こう、あなたの胸に手を当てて、\nすり、すり…❤すり…❤すり…❤\nお願いします、わたしたちを傷つけないでください❤,For example; by placing my hand and rubbing your chest... Rub...❤ Rub...❤ I beg of you; please don't hurt us.❤
こうやって乳首の辺りをゆっくり撫でられたら、気持ちいいでしょう？\n戦いのためではなく、安らぎを感じるために、\nあなたの体は使いませんか？たくさんお手伝いしてあげますよ…❤,Doesn't it feel good when I gently touch your nipples like this? Wouldn't you like to use your body not for fighting; but for relaxing and experiencing pleasure? I'm happy to help you with that...❤
そう、力を抜いて…❤\nわたしの手の平に、優しい快楽に身を任せて…❤\nいい顔ですね。もっともっと愛を感じて、蕩けてください…❤,Yes; just relax...❤ Allow yourself to give in to the gentle pleasure of my hand...❤ You have such a lovely expression. Let yourself indulge in my love even more; and drown in bliss...❤

> えっと、その、わたしのスカートの中、気になりますか？\nさきほどからわたしのふとももの辺りをチラチラ見ていますよね。\nスカートが揺れて、気になっているのかなぁ、なんて…。,Um; pardon me... Do you wish to peek under my skirt? You've been glancing at my thighs for a while now; so I was wondering if you might've been distracted by my skirt fluttering about...

Reviewer 2: Double 'skirt' feels kind of weird. Maybe the 'hem' fluttering about?

> えっと、その、わたしのスカートの中、気になりますか？\nさきほどからわたしのふとももの辺りをチラチラ見ていますよね。\nスカートが揺れて、気になっているのかなぁ、なんて…。,Um; pardon me... Do you wish to peek under my skirt? You've been glancing at my thighs for a while now; so I was wondering if you might've been distracted by my skirt swaying around...

Reviewer: ```suggestion
えっと、その、わたしのスカートの中、気になりますか？\nさきほどからわたしのふとももの辺りをチラチラ見ていますよね。\nスカートが揺れて、気になっているのかなぁ、なんて…。,Um; pardon me... Do you wish to peek under my skirt? You've been glancing at my thighs for a while now; so I was wondering if you might've been distracted by my skirt fluttering about...
```

あまりエッチなことはできませんし、\nこれで満足していただけるのであれば、ほら、ぴらっ❤\nパンツを見ても構わないので、攻撃しないでください❤,Though I can't do anything too lewd; if this much is enough to satisfy you; then here you go.❤ You may look at my panties as much as you'd like; just please don't attack me.❤

> そう、そのままそこで、じっとパンツを見ていていいんです。\n一歩でも近づいたら、スカート下ろして見えなくしちゃいますからね。\nくすくす❤足を止めちゃって、いい子ですね…❤,Yes; feel free to stand there and stare at my panties. But if you take even one step closer; then I will pull the hem back down; and you won't be able to see them anymore; understood? *Giggle*~. You stopped advancing; good boy.❤

Reviewer 2: ```suggestion
そう、そのままそこで、じっとパンツを見ていていいんです。\n一歩でも近づいたら、スカート下ろして見えなくしちゃいますからね。\nくすくす❤足を止めちゃって、いい子ですね…❤,Yes; feel free to stand there and stare at my panties. But if you take even one step closer; then I will pull the hem back down; and you won't be able to see them anymore; understood? *Giggle* You stopped advancing. Such a good boy.❤
```

> そう、そのままそこで、じっとパンツを見ていていいんです。\n一歩でも近づいたら、スカート下ろして見えなくしちゃいますからね。\nくすくす❤足を止めちゃって、いい子ですね…❤,Yes; feel free to stand there and stare at my panties. But if you take even one step closer; then I will pull the hem back down; and you won't be able to see them anymore; understood? *Giggle*~ You stopped advancing; what an obedient child.❤

Reviewer: ```suggestion
そう、そのままそこで、じっとパンツを見ていていいんです。\n一歩でも近づいたら、スカート下ろして見えなくしちゃいますからね。\nくすくす❤足を止めちゃって、いい子ですね…❤,Yes; feel free to stand there and stare at my panties. But if you take even one step closer; then I will pull the hem back down; and you won't be able to see them anymore; understood? *Giggle*~. You stopped advancing; good boy.❤
```

> 足で踏むなんて申し訳ないのですが、これも戦いですから。\nあなたの大事なところ、\n踏ませていただきますね。えいっ❤,I apologize for trampling you with my feet; but we are in battle; after all. I shall step over your sensitive parts now; okay? Hah!❤

Reviewer: ```suggestion
足で踏むなんて申し訳ないのですが、これも戦いですから。\nあなたの大事なところ、\n踏ませていただきますね。えいっ❤,I apologize for trampling your cock under my feet; but we are in battle; after all. I shall step over your sensitive parts now; okay? Hah!❤
```

> えと、強くは踏まずに軽くグニグニするだけなので、\n痛くはないと思います。どうですか、気持ちいいでしょうか…❤\nんっ、ぐにぐに…❤ふみ、ふみ…❤,Um; I'm trying to not press too hard and simply massage it a bit; so it shouldn't hurt too much. How do you feel? Does it feel good...?❤ Mm; rub rub...❤ Squish squish...❤

Reviewer 2: Small flow suggestion

```suggestion
えと、強くは踏まずに軽くグニグニするだけなので、\n痛くはないと思います。どうですか、気持ちいいでしょうか…❤\nんっ、ぐにぐに…❤ふみ、ふみ…❤,Um; I'm trying not to press too hard and simply massage it a bit; so it shouldn't hurt too much. How is it? Does it feel good...?❤ Mm; rub rub...❤ Squish squish...❤
```

> よかった喜んでいただけているようですね❤\n頬がゆるんで気持ちよさそうな顔❤\nうふふ❤その顔もっと見せてください。ほら、ぐにぐに❤すりすり…❤,You seem to be enjoying it; what a relief!❤ Your cheeks look flushed; and you have such a blissful expression.❤ Fufu~❤ Please; show me more of that blissful expression. Here; squish squish~❤ Rub rub...❤

Reviewer 2: Double 'blissful expression' back to back. Maybe something like 'Show me more of that face' or something like that?

> えっと、こういうのでも、気持ちよくなってくれますか？\n唇に人差し指を当てて、んっちゅ…❤\n愛を込めた投げキッスです❤,Um; then how about this? Would you like it if I placed my finger to my lips and... *kiss*❤ It's a kiss packed with my love.❤

Reviewer 2: Might be better to sushi type out 'mwah' or 'mchu' here, to avoid repeating the word kiss.

Author: my take on it is "mwah" and "chu" give the impression of two different kinds of kisses, so "mwah" isn't an accurate translation of "chu". that's why I went with a neutral *kiss*, but I'll defer to you and Gira's judgement on what's most appropriate here.

えっと、気持ちよくしてほしい、ですか？\nあらあら、もう試合に疲れてしまったんですね。\nいいですよ❤その疲れた体、わたしが癒してあげます❤,Um; you'd like me to make you feel good; you said? Oh dear; this match has worn you out so much; hasn't it? It's okay~❤ Let me soothe your weary body for you.❤

> くす、ズボンの中で漏らしてしまいましたね❤\n服の上から優しく撫でられて、うっとり幸せに射精❤\nとても気持ちよさそうな顔をされてますよ❤,*Giggle*; did you have an accident in your pants just now?❤ You happily ejaculated with a smile on your face; all from being gently stroked over your pants.❤ You look very pleased and satisfied~❤

Reviewer 2: Feel free to make changes, just wanted to leave a suggestion here. 

```suggestion
くす、ズボンの中で漏らしてしまいましたね❤\n服の上から優しく撫でられて、うっとり幸せに射精❤\nとても気持ちよさそうな顔をされてますよ❤,*Giggle* You finished in your pants just now; didn't you?❤ You moaned in pure bliss as you came; all from being gently stroked over your clothes.❤ You look absolutely satisfied~❤
```

> くす、ズボンの中で漏らしてしまいましたね❤\n服の上から優しく撫でられて、うっとり幸せに射精❤\nとても気持ちよさそうな顔をされてますよ❤,*Giggle*; did you have an accident in your pants just now?❤ Seems you blissfully ejaculated from being gently stroked over your pants.❤ You have such a happy and contented look on your face~❤

Reviewer: Nice job phrasing 漏らして as "an accident". Keeps the right nuance! 
'blissfully ejaculated' is a bit of a weird phrase we should try to avoid.
The latest sentence has a couple of extra ideas that should be removed.
```suggestion
くす、ズボンの中で漏らしてしまいましたね❤\n服の上から優しく撫でられて、うっとり幸せに射精❤\nとても気持ちよさそうな顔をされてますよ❤,*Giggle*; did you have an accident in your pants just now?❤ You happily ejaculated with a smile on your face; all from being gently stroked over your pants.❤ You look very pleased and satisfied~❤
```

乳首だけしか触ってないのに、\nこんなにたくさん出してしまったんですか？\n乳首がずいぶんと敏感なんですね❤くすくす❤,You ejaculated just from me touching your nipples? It seems your nipples are quite sensitive; aren't they?❤ *Giggle*❤

> きゃ、急に射精したからびっくりしちゃいました。\nもしかしてパンツを見せられただけでイってしまったんですか？\nくすくす❤エッチなことにだいぶ弱いんですね。かわいらしい❤,Eek! You surprised me; ejaculating so suddenly like that. Did you ejaculate just from seeing my panties? *Giggle*❤ It seems you're quite weak against naughty things; aren't you? How cute.❤

Reviewer 2: You have 'ejaculate' back to back.

> きゃ、急に射精したからびっくりしちゃいました。\nもしかしてパンツを見せられただけでイってしまったんですか？\nくすくす❤エッチなことにだいぶ弱いんですね。かわいらしい❤,Eek! You surprised me; ejaculating so suddenly like that. Did you ejaculate just from seeing my panties? *Giggle*❤ It seems you're quite weak against naughty things; aren't you? How cute.❤

Reviewer: 'naughty things' is a bit weird and weak for English:
```suggestion
きゃ、急に射精したからびっくりしちゃいました。\nもしかしてパンツを見せられただけでイってしまったんですか？\nくすくす❤エッチなことにだいぶ弱いんですね。かわいらしい❤,Eek! You surprised me; ejaculating so suddenly like that. Did you ejaculate just from seeing my panties? *Giggle*❤ You get really weak when it comes to anything sexual; don't you? How cute.❤
```

Author: I tried to translate Cleric's lines with the lens of her being more "pure-hearted", so she'd use softer language and more euphemisms to refer to sexual stuff.

> うふ、ブーツの下でびゅくびゅく出ちゃってます❤\n優しく踏んでもらえて気持ちよかったですか？,*Giggle*; it's peeking out from beneath my boots.❤ Does it feel that good; being so tenderly stepped on like this?

Reviewer: Give this sentence another go. 
1. Try to translate the first sentence accurately
2. improve the flow for the second sentence so it's natural English

> くす❤目にハートを浮かべながら射精して、\nそんなにわたしの投げキスがよかったのでしょうか。\nあなたのこと幸せにできてうれしいです❤,*Giggle*❤ You ejaculated with such a lovestruck look in your eyes. Did me blowing a kiss make you that happy? That makes me so happy too!❤

Reviewer 2: I think you can delete the 'so' at the end to help with flow

> くす❤目にハートを浮かべながら射精して、\nそんなにわたしの投げキスがよかったのでしょうか。\nあなたのこと幸せにできてうれしいです❤,*Giggle*❤ You ejaculated with such a lovestruck look in your eyes. Did me blowing a kiss make you that happy? That makes me so happy too!❤

Reviewer: "with such a lovestruck look" is a bit of a weird English Phrase...
"me blowing a kiss" is a bit weird too

```suggestion
くす❤目にハートを浮かべながら射精して、\nそんなにわたしの投げキスがよかったのでしょうか。\nあなたのこと幸せにできてうれしいです❤,*Giggle*❤You ejaculated while your eyes went all heart-shaped! Did you enjoy recieving a blown kiss from me that much? That makes me so happy too!❤
```

うっ、ううっ・・・❤,Ngh; hnnnngh...❤❤

> あら？うずくまってどうされたのですか？\n暴力的な攻撃はしていないはずですが、\nどこか体をぶつけてしまったとか・・・。,My? Why are you bending over like that? I don't think I attacked you too violently; did you perhaps bump into something...? 

Reviewer: too violently is a bit weird
I think she's also talking about being hit hard generally, not just her doing it.
あら is more like an Oh? or huh?

```suggestion
あら？うずくまってどうされたのですか？\n暴力的な攻撃はしていないはずですが、\nどこか体をぶつけてしまったとか・・・。,Huh? Why are you bending over like that? You shouldn't have been hit to hard... Did you bump yourself in someway or something...? 
```

> いえ、よく見るとうずくまっているだけでなく、\n手が小刻みに揺れていますね。\nふふ、苦しいわけではなく自慰をしているだけのようです❤,No; looking closely; you're not just hunched over. Your arms are rustling around; too. Fufu; so it's not that you're hurt; but that you're masturbating.❤

Reviewer: ```suggestion
いえ、よく見るとうずくまっているだけでなく、\n手が小刻みに揺れていますね。\nふふ、苦しいわけではなく自慰をしているだけのようです❤,That's not it. On closer look; I can see you're not just hunched over; your arms are rustling around; too. Fufu; so it's not that you're hurt; but that you're masturbating.❤
```

快感に抗って戦うのが辛くなってしまいましたか？\nそれなら、ほら、こうしてあげます❤,Have you grown weary of fighting me and resisting your sexual urges? In that case; here; let me do something for you.❤
ぎゅっとあなたの頭を抱きしめて\nわたしの体温を感じながら、シコシコ続けてください❤,I'll hold your head tightly against my chest; so just keep masturbating as you feel the heat of my body against your skin.❤
オナニーに夢中で無防備なあなたをわたしが守ってあげます❤\n胸は大きくありませんが、\nこうしているだけで少しは安心できるでしょう？,I'll protect you while you're vulnerable and completely engrossed in masturbating.❤ My chest isn't very big; but you should feel safer like this; right?
うふふ❤どんどん体の力が抜けていきますね❤\nでも、しこしこする手は止まらない❤\nだらしないお顔になりながら、しこしこしこ❤しこしこしこ❤,Fufu~❤ Your body is feeling weaker and weaker; isn't it?❤ Yet your hand refuses to stop stroking.❤ You have such a blissful expression on your face as you keep stroking;❤ and stroking.❤

> 気持ちいいのとあったかいのに包まれて、\n何も考えられなくなりますか？それでいいんですよ❤\nムリして我慢なんてしなくていいんです❤,When you're wrapped in such a warm and pleasant sensation; it's hard to think about anything else; isn't it? It's alright~❤ There's no need to force yourself if you can't.❤

Reviewer 3: ```suggestion
気持ちいいのとあったかいのに包まれて、\n何も考えられなくなりますか？それでいいんですよ❤\nムリして我慢なんてしなくていいんです❤,When you feel nice and warm like this; it's hard to think about anything else; isn't it? It's alright~❤ There's no need to force yourself if you can't.❤
```

ううっ、あうう・・・❤,Ugh; uwaah...❤
そうそう。しこしこ上手ですね❤\n上手にできたえらい子は、頭を後ろからなでてあげますからね❤,Yes; that's it. You're stroking your thing so well; aren't you?❤ For such a good boy who's working so hard; I'll give you a pat on the head; okay?❤

> 息も大きく吸って、リラックスして…❤\nわたしの体、いい匂いがしますか？\nふふ、そう感じてもらえるならうれしいです❤,Just take a deep breath; and relax...❤ Do I smell good? Fufu; I'm happy if you think so.❤

Reviewer: ```suggestion
息も大きく吸って、リラックスして…❤\nわたしの体、いい匂いがしますか？\nふふ、そう感じてもらえるならうれしいです❤,Just take a deep breath; and relax...❤ Do I smell good? Fufu; I'm happy you think so.❤
```

しこしこ、しこしこ、イクまでゆっくり扱いていいですからね❤\nあなたの我慢はもうおしまい。\nあとは快感に身をゆだねて、ただただ楽しみましょう❤,Stroke stroke; stroke stroke~ Just take it slow; until you're ready to cum.❤ It seems you're about to reach your limit; so just give in to the pleasure and let it wash over you.❤ 
あら、体が震えて・・・❤\nもうすぐ出しちゃうんですね❤\nふふ、我慢はしなくていいですよ❤,Oh my; your body is trembling...❤ You're on the verge of ejaculating; aren't you?❤ Fufu; you don't need to hold back.❤

> しこしこ❤しこしこ❤しこしこ❤\nはい、気持ちよく出しましょうね❤びゅ、びゅっー❤,Stroke stroke~❤ Stroke stroke~❤ Stroke stroke~❤ Okay; let's indulge ourselves and let it all out; shall we?❤ Pew; pew~❤

Reviewer 2: ```suggestion
しこしこ❤しこしこ❤しこしこ❤\nはい、気持ちよく出しましょうね❤びゅ、びゅっー❤,Stroke stroke~❤ Stroke stroke~❤ Stroke stroke~❤ That's it. Indulge yourself and let it all out; okay?❤ Pew; pew~❤
```

はひぃ・・・❤,Haaaah...❤
うふふ❤たくさん出せましたね❤\n戦いを捨て、快感に身を任せることができて、\nとてもいい子でしたよ❤えらい❤えらい❤,Fufu~❤ You managed to release so much; didn't you?❤ You're such a brave child; choosing to stop fighting and accept the feeling of pleasure.❤ Good boy.❤ You're such a good boy.❤ 

> あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n二人とも幸せになれて、素晴らしい結果ですね❤ふふ❤,And thanks to your energy; now I've recovered as well. You were healed with pleasure; and I was healed with your energy... What a wonderful; mutually beneficial outcome; right?❤ Fufu~❤

Reviewer 2: I think 'don't you think' would hit harder than 'right'

```suggestion
あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n二人とも幸せになれて、素晴らしい結果ですね❤ふふ❤,And thanks to your energy; now I've recovered as well. You were healed with pleasure; and I was healed with your energy... What a wonderful; mutually beneficial outcome; don't you think?❤ Fufu~❤
```

> あら？うずくまってどうされたのですか？\nもしかしてもあなたも・・・❤,My? Why are you bending over like that? Could it be that you also...?❤

Reviewer: ```suggestion
あら？うずくまってどうされたのですか？\nもしかしてもあなたも・・・❤,Oh? Why are you bending over like that? Could it be that you also...?❤
```

ふふ、やはり手が小刻みに揺れていますね❤\n苦しいわけではなく自慰をしているだけのようです❤\n仲間の方が快楽に身を任せたのを見て、うらやましくなりましたか？,Fufu; your arms are rustling around; aren't they?❤ It seems you're not in pain; but simply masturbating instead.❤ Were you envious; seeing your ally give in to pleasure?
もう戦うのが辛いのですよね。\nそれなら、ほら、こうしてあげます❤,It's hard to keep fighting like that; isn't it? In that case; here; let me do something for you.❤
ぎゅっとあなたの頭を抱きしめて、\nわたしの体温を感じながら自慰を続けてください❤,I'll hold your head tightly against my chest; so just keep masturbating as you feel the heat of my body against your skin.❤
胸はほとんどないんですけども、\nこうしているだけで少しは安心できるでしょう？\nあなたを傷つける人間はいませんよ。わたしが守ってあげます❤,My chest is quite modest; but just being like this makes you feel a bit safer; doesn't it? I'll protect you; so no harm will befall you.❤

> そうそう。しこしこ上手ですね❤\nあなた心安らかになれるように、\n頭を後ろからなでてあげますからね❤,Yes; that's it. You're stroking your thing so well; aren't you?❤ Let me stroke your head to help soothe you; okay?❤

Reviewer 2: If recommend bringing this more in line with your translation on 38 for consistency. Which one you use as the template is your call, though.

息を大きく吸って、リラックスして…❤\nわたしの体、いい匂いがしますか？\nふふ、そう感じてもらえるならうれしいです❤,Just take a deep breath; and relax...❤ Do I smell good? Fufu; I'm happy to hear it.❤

> ほら、わたしの体に身を預けて、\nしこしこ❤しこしこ❤\nはい、気持ちよく出しましょうね❤びゅ、びゅっー❤,Come on; press your body closer to mine. Stroke stroke~❤ Stroke stroke~❤ Okay; let it all out in one big; satisfying release!❤ Pew; pew~❤

Reviewer 2: ```suggestion
ほら、わたしの体に身を預けて、\nしこしこ❤しこしこ❤\nはい、気持ちよく出しましょうね❤びゅ、びゅっー❤,Come on; press your body closer to mine. Stroke stroke~❤ Stroke stroke~❤ Okay; now let it all out in one big; satisfying release!❤ Pew; pew~❤
```

うふふ❤たくさん出せましたね❤\n戦いを捨て、快感に身を増せることができて、\nとてもいい子でしたよ❤えらい❤えらい❤,Fufu~❤ You released so much; didn't you?❤ You're such a good boy; giving up on fighting to indulge in pleasure instead.❤ Good boy.❤ Such a good boy.❤

> あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n幸せで素晴らしい結果ですね❤,I've recovered too; thanks to your energy. You were healed through pleasure; and I was healed with your energy. It's an outcome that leaves everyone happy; isn't it~❤

Reviewer 2: ```suggestion
あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n幸せで素晴らしい結果ですね❤,I've recovered too; thanks to your energy. You were healed through pleasure; and I was healed with your energy. It's an outcome that leaves everyone happy; don't you think?❤
```

あううっ、だめですよ…。そんな暴力的なことばかりしちゃ…。\nねえ、武器なんて捨てて、お互いに癒しを与え合いませんか？,Owww; that's no good... You shouldn't be doing such violent things... Hey; why don't we lay down our weapons and try soothing each other instead?
例えばほら、わたしの膝の上に頭を乗せるのはどうでしょう。\nここに身をゆだねてくだされば、\n優しく頭をなでながら、あなたのこと気持ちよくしてあげますよ❤,For example; why don't you rest your head on my lap? If you'll allow me to; then I'll gently stroke your head and make you feel good.❤

> もちろん、その大きくしてしまった男性器も、\nもう片方の手で優しく扱いてあげます。\n膝枕されながらの手での奉仕、きっと素敵な体験になりますよね❤,Naturally; I'll also use my other hand to caress that embiggened penis. Being serviced like that; while your head rests on my lap; would surely feel wonderful; wouldn't it?❤

Reviewer: embiggened penis, Being serviced like that; are very weird for an English speaker to say.
```suggestion
もちろん、その大きくしてしまった男性器も、\nもう片方の手で優しく扱いてあげます。\n膝枕されながらの手での奉仕、きっと素敵な体験になりますよね❤,Of course; I'll also use my other hand to caress that swollen penis of yours. When you're being serviced like that while your head rests on my lap; it would surely feel wonderful; wouldn't it?❤
```

Author: I did consider swollen, but idk. Swollen kind of sounds like the penis is so big because it's damaged rather than erect.

Reviewer 2: If you want a more natural, still fancy word, 'engorged' is always an option.

> さあ、わたしに愛撫されるのを想像してください❤\n体重をあずけた状態でしこしこしこしこ❤\nね、気持ちよさそうでしょう？,Picture it; my hand caressing you;❤ rubbing and stroking your thing as you lean into me.❤ Doesn't that sound absolutely blissful?

Reviewer 2: Lowercase letters after a heart always look weird to me. Maybe this?

```suggestion
さあ、わたしに愛撫されるのを想像してください❤\n体重をあずけた状態でしこしこしこしこ❤\nね、気持ちよさそうでしょう？,Picture it; my hand caressing you~❤ Rubbing and stroking your thing as you lean into me.❤ Doesn't that sound absolutely blissful?
```

> 遠慮せずこちらにどうぞ❤\nわたしのお膝の上に、頭を置いて楽にしてくださいね❤,Don't hold yourself back; come here.❤ Rest your head upon my lap and make yourself comfortable; okay?❤

Reviewer 2: I think 'there's no need to hold back' might sound better.

> 理性を振りしぼって攻撃した,No Rest for the Wicked

Reviewer: While I appreciate the effort for this sentence, we should translate it faithfully.

```suggestion
理性を振りしぼって攻撃した,Muster up your willpower and attack
```

> 彼女に身を任せてしまった,'These weary bones of mine...'

Reviewer: This doesn't make sense for this context. Both translation wise and English wise.
```suggestion
彼女に身を任せてしまった,Leave everything to her...
```

いいこ、いいこ…❤戦うのをやめられて偉いですね…❤\nそう、力を抜いて…❤\nリラックスしているのは、とてもいいことなんですよ…❤,Sweet child; you're such a good boy...❤ It was brave of you to stop fighting...❤ Yes; just relax...❤ Resting feels wonderful; doesn't it...?❤
互いに傷つけあうのは悪いこと…❤\n傷つけないでボーっとしてるのは、善いこと…❤\nあなたはいま善いことをしてるんです…❤,Hurting each other is wrong...❤ It's better to just stop and smell the roses together...❤ You're doing a good thing right now...❤
わたしの声を聴いていると力が抜けてきますね…❤\n頭がぼーっとして、気持ちよくなる…❤\nこの気持ちいいのは善いことをしたあなたへのご褒美…❤,Listening to my voice feels soothing; doesn't it...?❤ Your mind feels hazy; bathed in pleasure...❤ This blissful feeling is a reward for your good choices...❤
だから身を任せていいんです…❤\n何も考えなくていいんです…❤\nふふ、力が抜けてだらーんとしてきちゃいますね❤えらい、えらい❤,So you can just let yourself go...❤ You don't have to think about anything...❤ Fufu; your body has relaxed and released all your tension; hasn't it?❤ That's good; very good.❤
うあ、ああっ・・・❤,Uwa; aah...❤
あらあら、うふふ❤\n膝枕をするまえに、お耳にささやかれただけで出してしまったのですか？\nくす❤ちょっと力が抜けすぎちゃいましたね❤,Oh; oh my; fufu~❤ Did you ejaculate just from me whispering in your ear; before I even had the chance to give you a lap pillow? *Giggle*❤ You let yourself go a bit too much; didn't you?❤

> 我慢してたおちんちんから、とろとろーって気持ちよさそうなお漏らし❤\nでも、ちゃんと出せて偉いですよ❤\n武器を捨てて射精もできて、とってもいい子❤えらいえらい❤,It seems like there's a pleasurable stream of pee leaking out of your pent-up penis.❤ Still; you did a really good job letting it all out properly.❤ You're such a good boy; tossing your weapon aside and ejaculating.❤ Good boy; such a good boy.❤

Reviewer: 'pee' is absolutely the wrong word to use here. Have another crack at the first sentence here.

> 我慢してたおちんちんから、とろとろーって気持ちよさそうなお漏らし❤\nでも、ちゃんと出せて偉いですよ❤\n武器を捨てて射精もできて、とってもいい子❤えらいえらい❤,You've held back for so long; and now there's so much semen leaking out of your pent-up dick.❤ But it's great that you were able to let it out properly.❤ You're such a good boy; putting down your weapon and managing to ejaculate.❤ Good boy; such a good boy.❤

Reviewer 2: 'Penis' might be as vulgar as I go with this charcter. 'Dick' feels off

Author: I agree, but 'pent-up penis' sounds too alliterative for my tastes

Reviewer 2: That's fair. What about 'neglected' or 'overteased'?

Reviewer 2: Or you could change dick to 'little guy' or 'little thing' or something

ああ・・❤あううっ・・・❤,Aah...❤ Hnngh...❤

> うふふ❤もう立っていられませんか？\nそう膝の上に頭をゆっくり乗せてください❤\n頭を置いたら、考えることはおしまい❤あとは気持ちいいことだけ❤,Fufu~❤ Is it hard to stay on your feet? Go ahead and rest your head gently on my lap.❤ Once you're nice and comfortable on my lap; you can stop thinking about anything else.❤ Just focus on feeling good.❤

Reviewer: lap is repeated

> ほら、わたしのふとももの温かさ、\nほっぺで感じて気持ちいいですね❤\n体がリラックスして頭がふわふわ❤意識もとろーん❤,There; doesn't the warmth of my thighs feel pleasant against your cheek?❤ Your body is relaxed; your head feels light and fluffy; and your consciousness is drifting slowly away...❤

Reviewer: First sentence is a bit unnatural. Maybe something like this?

```suggestion
ほら、わたしのふとももの温かさ、\nほっぺで感じて気持ちいいですね❤\n体がリラックスして頭がふわふわ❤意識もとろーん❤,There; the warmth of my thighs squeezing your cheeks feels good; doesn't it?❤ Your body is relaxed; your head feels light and fluffy; and your consciousness is slowly drifting away...❤
```

> ほら、わたしのふとももの温かさ、\nほっぺで感じて気持ちいいですね❤\n体がリラックスして頭がふわふわ❤意識もとろーん❤,There; the warmth of my thighs squeezing your cheeks feels good; doesn't it?❤ Your body is relaxed; your head feels light and fluffy; and your consciousness is slowly drifting away...❤

Reviewer 2: 'Squeezing your cheeks' is probably wrong. Her image has him laying sideways. 'Pressing against' would likely be more accurate.

> 清潔でいい匂いがして、しあわせ、しあわせ…❤\nおちんちんもしこしこしこ❤,This pure and sweet-smelling sensation; pure happiness; pure bliss...❤ as you keep stroking your penis.❤

Reviewer 2: ```suggestion
清潔でいい匂いがして、しあわせ、しあわせ…❤\nおちんちんもしこしこしこ❤,Let the pure and sweet-smelling sensation leave you in utter bliss...❤ And I'll just keep stroking your penis~❤
```

あっ、ああっ・・・・❤\nきもちいい・・・❤,Ah; aah...❤ It feels so good...❤

> うふふ、素直に「きもちいい」って言えてえらいですね❤\n素直に甘えられるのは、とてもいいことなんですよ❤,Fufu~ Being able to honestly say it feels good; you're such a good boy; aren't you?❤ Being able to openly express your feelings and let yourself be pampered is a wonderful thing.❤

Reviewer 2: Small change to cut down on repetition

```suggestion
うふふ、素直に「きもちいい」って言えてえらいですね❤\n素直に甘えられるのは、とてもいいことなんですよ❤,Fufu; to so honestly say how good it feels... You're such a good boy; aren't you?❤ Being able to openly express your feelings and let yourself be pampered is a wonderful thing.❤
```

> 隠しごとや嘘をつくことは悪いこと…❤\n素直に甘えられるのは善いこと…❤\nおちんちん気持ちいいって言えて、えらい、えらい…❤,Keeping secrets and lying is sinful...❤ Honesty and affection is virtue...❤ Being able to say that your penis feels good; you're such a good boy...❤

Reviewer: "you're such a good boy" doesn't link with "Being able to say that your penis feels good; "

> 褒められると頭がぽわってあったかくなりますね❤\nもっと褒めてほしいって体がうずいちゃいますね。\nほら、それならたくさん「きもちいい」って言いましょう？,Being praised makes you feel warm and fuzzy inside; doesn't it?❤ Your spine gets all tingly; making you want even more praise; right?❤ Come on; how about declaring "it feels good" even more? 

Reviewer: "how about declaring "it feels good" even more?" is unusual English

あなたがいい子になれるように、おちんちんなでなでしてあげます❤\n気持ちいいのでいっぱいにしてあげます❤しこしこしこ❤,I'll help motivate you to be good by stroking your penis for you.❤ I'll make you feel so; so good.❤ Stroke stroke stroke~❤
うう、きもちいい・・・❤きもちいい・・・❤\nきもちいい・・・❤❤ああっ、もっとなでて・・・❤\nきもちいいっ・・・❤❤,Uwaah; it feels good...❤ It feels so good...❤ It feels goooood...❤❤
いいですよ❤ほら、えらい❤えらい❤\n頭もおちんちんもいっぱいなでてあげます❤,There's a good boy.❤ See; good job.❤ You did so well.❤
たくさんなでてあげてるのに、\nおちんちん、ひくひくして\nもっとなでてぇっておねだりしちゃってますね❤,I'm already stroking you; yet your little penis is still twitching so much; begging for more; huh?❤

> 欲しがりでかわいいおちんちん❤\n甘え上手でえらい、えらい❤\n気持ちよくなれて、えらい、えらい❤,What cute little eager penis.❤ You're so good at asking for love; amazing.❤ You already know how to let yourself feel good; so wonderful. You're such a good child.❤ 

Reviewer: ```suggestion
欲しがりでかわいいおちんちん❤\n甘え上手でえらい、えらい❤\n気持ちよくなれて、えらい、えらい❤,What cute little eager penis.❤ You're so good at asking for love; amazing.❤ You already know how to let yourself feel good; so wonderful. You're such a good boy.❤ 
```

> 欲しがりでかわいいおちんちん❤\n甘え上手でえらい、えらい❤\n気持ちよくなれて、えらい、えらい❤,What cute little eager penis.❤ You're so good at asking for love; amazing.❤ You already know how to let yourself feel good; so wonderful. You're such a good boy.❤ 

Reviewer 2: Felt weird. Here's a suggestion. 

```suggestion
欲しがりでかわいいおちんちん❤\n甘え上手でえらい、えらい❤\n気持ちよくなれて、えらい、えらい❤,What a cute; eager little penis.❤ You're so good at letting me pamper you. It's wonderful. ❤ You already know how to let yourself feel good. Good boy. You're such a good boy~❤ 
```

しこしこ、しこしこ❤\nあら、むずむず腰を動かして、\nもうすぐ出しちゃいそうなんですか？,Stroke stroke; stroke stroke~❤ Oh my; your hips are squirming so restlessly. Are you ready to ejaculate?
我慢はしなくていいですよ❤\n力を抜いて、気持ちいいのが昇ってくるのに任せて、\nほら、お漏らししてしまいましょう❤,You don't have to hold yourself back.❤ Just relax and let it out. Let yourself be baptized in this feeling of pleasure. Go on; grant yourself this release.❤

> わたしの手になでてもらいながら、\nおちんちんからびゅっ、びゅー❤\nリラックスして、頭が真っ白になるぐらい気持ちよくなりましょうね❤,Shoot it out of your penis as I stroke you with my hand. Pew~❤ Just relax; and let yourself feel good until your mind goes numb; okay?❤

Reviewer: ```suggestion
わたしの手になでてもらいながら、\nおちんちんからびゅっ、びゅー❤\nリラックスして、頭が真っ白になるぐらい気持ちよくなりましょうね❤,Shoot it all out of your penis as I stroke you with my hand. Pew~❤ Just relax; and let yourself feel good until your mind goes numb; okay?❤
```

はひぃっ・・・❤きもちいい・・・❤❤,Hwaah...❤ It feels so good...❤

> うふふ❤イったあとも気持ちいいって言えてえらいですね❤\n全部出し切るまで、やさしくなでてあげますね❤,Fufu~❤ You're such a good boy; aren't you? You managed to say "it feels good" even after you climaxed.❤ I'll keep gently caressing you until you've let it all out; okay?❤

Reviewer: It's not clear why saying "it feels good"  is linked to being a good boy: Maybe like this would be better?
```suggestion
うふふ❤イったあとも気持ちいいって言えてえらいですね❤\n全部出し切るまで、やさしくなでてあげますね❤,Fufu~❤ You're such a good boy; aren't you? You were able to keep telling me how good you're feeling even after you climaxed.❤ I'll keep gently caressing you until you've let it all out; okay?❤
```

> 戦いをやめることができて、いいこ、いいこ❤\nおちんちんから射精もできて、とってもえらい❤えらい❤\nうふ、幸せそうな顔です❤,You chose to stop fighting; you're such a brave child; such a good boy!❤ You also managed to let yourself ejaculate; so wonderful!❤ You're amazing.❤ Fufu; you look so happy.❤

Reviewer 2: ```suggestion
戦いをやめることができて、いいこ、いいこ❤\nおちんちんから射精もできて、とってもえらい❤えらい❤\nうふ、幸せそうな顔です❤, It was so brave of you to stop fighting. fighting. You're such a good boy. ❤ You even managed to let yourself ejaculate. It was wonderful.❤ You're amazing.❤ Fufu; you look so happy~❤
```

> オナニーして敗北、気持ちよかったですか？\n昇格試合でしたが、気に病むことはありません。\n男の子が女性に甘えるのは、当然のことですから❤,You masturbated and lost; but did it feel good? This may be a promotion match; but don't worry yourself over it. It's only natural a boy would want to be pampered by a woman.❤

Reviewer 2: ```suggestion
オナニーして敗北、気持ちよかったですか？\n昇格試合でしたが、気に病むことはありません。\n男の子が女性に甘えるのは、当然のことですから❤, Did throwing the match to masturbate feel good? I know it was a promotion match; but don't worry yourself over it too much. It's only natural a boy would want to be pampered by a woman.❤
```

> 暴力をふるうよりもずっとよいことができましたね❤えらいですよ❤,You rose beyond resorting to such senseless violence; didn't you?❤ That's amazing.❤

Reviewer: I can see where you're going with this, but 'You rose beyond' is a little weird. Try re-writing it

> 暴力をふるうよりもずっとよいことができましたね❤えらいですよ❤,You chose to be above such senseless violence; didn't you?❤ That's amazing.❤

Reviewer 2: Personal preference but I think 'so wonderful' would carry her personality better than 'amazing'

> あら、ズボンの中でびゅくびゅく脈打っています❤\nわたしがなでたことで、すっかり楽になれたみたいですね。\nうふふ❤,Oh my; it's pulsing and throbbing inside your pants!❤ It seems you've really perked up once I started stroking it; haven't you? Fufu~❤

Reviewer 2: ```suggestion
あら、ズボンの中でびゅくびゅく脈打っています❤\nわたしがなでたことで、すっかり楽になれたみたいですね。\nうふふ❤,Oh my; it's pulsing and throbbing inside your pants!❤ It seems you really perked up once I started stroking it; didn't you? Fufu~❤
```

あら、なにやら腰がビクビクして・・・❤\nくす❤乳首が気持ちよくてイってしまったんですか？\n弱くてかわいらしいマスターさんですね❤,Oh my; your hips are twitching...❤ Fufu; did your nipples feel so good that you ejaculated? You're such a weak and adorable master; aren't you?❤
きゃ、急に射精したからびっくりしちゃいました。\nもしかしてパンツを見せられただけでイってしまったんですか？\nくすくす❤ずいぶんうぶなマスターさんなんですね❤,Eep! You surprised me; suddenly ejaculating like that. Did you get excited just from seeing my underwear? *Giggle*❤ You're quite the innocent master; aren't you?❤

> あん、ブーツの下でびゅくびゅく出ちゃってます❤\nきちんと気持ちよくできたみたいですね❤\n満足してもらえてうれしいです❤,Ah; it's gushing out from beneath my boots.❤ It seems you really enjoyed that; didn't you?❤ I'm happy you were so satisfied.❤

Reviewer: ```suggestion
あん、ブーツの下でびゅくびゅく出ちゃってます❤\nきちんと気持ちよくできたみたいですね❤\n満足してもらえてうれしいです❤,Ah; it's gushing out from under my boot.❤ It seems you really enjoyed that; didn't you?❤ I'm happy you were so satisfied.❤
```

きゃ、急に射精したからびっくりしちゃいました。\n投げキスしてあげるだけで射精しちゃうなんて、くすくす❤\nずいぶんうぶなマスターさんなんですね❤,Eep! You surprised me; ejaculating so suddenly. To think you'd cum just from me blowing you a kiss; *giggle*❤ You're quite the innocent master; aren't you?❤
わたしは可愛くて好きですよ❤\nほら、もう一回、んっちゅ❤\n幸せに包まれてください❤,I love that cute side of you.❤ Come on; show it to me again. *Kiss*❤ May you be blessed with happiness.❤

> {自分}にエナジーを搾り取られた僕は、体力を失い、闘技場の地面に膝をついていた。,With my energy drained by {自分}; I lost my strength and collapse to the arena floor.

Reviewer: ```suggestion
{自分}にエナジーを搾り取られた僕は、体力を失い、闘技場の地面に膝をついていた。,With my energy drained by {自分}; I lost my strength and collapsed on the arena floor.
```

> うふふ❤試合お疲れさまでした。\n気持ちよくなることができて、よかったですね❤,Fufu~❤ It was a good match. It's great you got to feel so good; isn't it?❤

Reviewer 2: What do you think about this translation instead?

```suggestion
うふふ❤試合お疲れさまでした。\n気持ちよくなることができて、よかったですね❤,Fufu~❤ That was a wonderful match.  I'm so glad you got the chance to feel good.❤
```

> わたしはあくまで味方のサポートが仕事で、\n直接エッチなことはあまりしないのですが、\nマスターさんを射精させられるなんて、びっくりです❤,My role is primarily to support my allies; and I rarely ever engage in direct sexual acts; but I'm surprised I managed to make Sir Master feel so good.❤

Reviewer: 'sir master' is a bit weird

Author: yeah, I really don't know how to translate マスターさん. to me, Mister Master sounds weird, but I really don't know how to naturally translate it

Reviewer 2: Worst case you could just go with 'you.' Sometimes it's necessary too keep it simple for localization

Author: Okay, I tried something out

> ずっと我慢をしていたんですよね❤\nわたしに抱きしめられながら、シコシコと自慰をなさって、\nふふ、すっきりするまで射精できましたか？,You've been pent up for so long; haven't you?❤ You were stroking so much while I hugged you. Fufu; were you able to ejaculate until you were satisfied?

Reviewer 2: That last sentence felt a little off. What do you think of this?

```suggestion
ずっと我慢をしていたんですよね❤\nわたしに抱きしめられながら、シコシコと自慰をなさって、\nふふ、すっきりするまで射精できましたか？,You've been pent up for so long; haven't you?❤ You were stroking so much while I hugged you. Fufu; was your orgasm enough to finally satisfy you?
```

あううぅ・・・❤,Hwaaaah...❤

> くすくす❤甘やかされてすっかり腑抜けちゃって、\n無敗のルーキーくんは甘えん坊なのね。,*Giggle*❤ So weak-willed and spoiled; the Undefeated Rookie turned out to be quite needy; hm?

Reviewer: Is needy the right word here?

Author: I can try some other words depending on the nuance you wanna go for. I went with needy here, but I could also try clingy, affectionate, dependent, etc.

これまでずっと我慢して頑張ってきたんでしょう？\nそれが優しく甘やかされて緊張の糸が切れてしまった。\n年頃の男の子だもの。そうなるのも仕方ないのかも。,You've been enduring and working hard all this time; haven't you? But the moment you received some kindness and pampering; your stoic mask finally broke. You're still a young boy; after all. It's only natural that things turned out this way.
わたしの言う通り暴力を捨てることができて偉かったですよ❤\nただ囁いているだけで射精してしまいましたが、\nお耳が弱いんですか？うふふ❤,You did well; giving up on violence when I asked you to.❤ Though you also ejaculated just from me whispering in your ears. Are they a sensitive area for you? Fufu~❤

> わたしの言う通り暴力を捨てることができて偉かったですよ❤\n膝枕をしながらの手での奉仕、\n蕩けた顔になっていて大変可愛かったです❤,You did well; giving up on violence when I asked you to.❤ Your face was so sweet and adorable when you were resting your head on my lap as I stroked your penis.

Reviewer: Maybe try different descriptions other than "sweet and adorable". it doesn't quite fit the Japanese

うう、あううぅ・・・❤,Ngh; hnnngh...❤

> くすくす❤すっかり腑抜けちゃって。\nそんなに強い攻撃じゃなかったでしょうに。\n無敗のルーキーくんは甘やかされるのに弱いのかしら。,*Giggle*❤ You've completely lost your composure. That attack wasn't very strong; was it? Is the Undefeated Rookie perhaps weak to being coddled?

Reviewer 2: The second line as a question doesnt seem to fit the original, but I like what you're going for. How about this?

```suggestion
くすくす❤すっかり腑抜けちゃって。\nそんなに強い攻撃じゃなかったでしょうに。\n無敗のルーキーくんは甘やかされるのに弱いのかしら。,*Giggle*❤ You've completely lost your composure. That attack wasn't very strong; you know.  Is the Undefeated Rookie perhaps weak to being coddled?
```

> これまでずっと我慢して戦ってきた分\n優しくされて緊張の糸が切れてしまった。\n年頃の男の子だもの。いずれそうなるのも仕方ないわよね。,After enduring and fighting for so long; a bit of pampering causes all your pent-up desire to spill out. You're just a young boy; after all; something like this was bound to happen.

Reviewer 3: ```suggestion
これまでずっと我慢して戦ってきた分\n優しくされて緊張の糸が切れてしまった。\n年頃の男の子だもの。いずれそうなるのも仕方ないわよね。,After enduring and fighting for so long; a bit of pampering causes all your pent-up desires to spill out. You're just a young boy; after all; something like this was bound to happen.
```

> どう？イったばかりだけど立てそう？\n立てるのなら、試合を終わりにして、観客に手でも振って退場しましょう。\nエナジーを出したばかりだから、まだ難しいかしら。,How do you feel? You just ejaculated; but do you think you can stand? If you can; then let's end this match; wave to the audience; and exit the arena. Though since you just released so much energy; it might be difficult right now.

Reviewer 2: I think this little edition helps the flow

```suggestion
どう？イったばかりだけど立てそう？\n立てるのなら、試合を終わりにして、観客に手でも振って退場しましょう。\nエナジーを出したばかりだから、まだ難しいかしら。,How do you feel? I know you just ejaculated; but do you think you can stand? If you can; then let's end this match; wave to the audience; and exit the arena. Though since you just released so much energy; it might be difficult right now.
```

> 僕はシルヴィアさんにそう言われて、なんとか立ち上がろうとする。しかし、足がぷるぷると震えてしまい、力が入らない。快感がまだ残っていて、腰がじんじんとうずいている。,At Sylvia's words; I tried to stand. However; my legs were trembling uncontrollably; and I couldn't put any weight on them. The feeling of pleasure still lingered; and my hips were numb.

Reviewer: This whole line is a bit weird. Try to make it more natural

あわわ、ダメですよ、無理をしたら。\nもし倒れたら大変です。,Oh dear; that's not good. You shouldn't push yourself too hard. It'd be terrible if you were to fall over.

> そう言って{自分}は僕のもとにかけより、体を抱きとめて倒れないように支えてくれた。ぽふん、と僕の顔が彼女の胸に当たる形になる。,{自分} rushes over and catches me; supporting me so I wouldn't fall over. My face landed softly against her chest.

Reviewer 2: Double 'me' feels weird. What do you think?

```suggestion
そう言って{自分}は僕のもとにかけより、体を抱きとめて倒れないように支えてくれた。ぽふん、と僕の顔が彼女の胸に当たる形になる。,{自分} rushes over and catches me; supporting my weight so I wouldn't fall over. My face landed softly against her chest.
```

> {自分}の胸は決して大きくなくて、ほんのりと膨らんでいるぐらいなのだが、それでも柔らかく、甘い匂いがする。僕はその感触と匂いでさらに力が抜けて、彼女にぎゅっと抱き着いてしまった。,{自分}'s chest wasn't especially large; just a slight protrusion; but it was soft and had a sweet scent. This feeling and scent made me even weaker; and I ended up clinging to her more tightly.

Reviewer: ```suggestion
{自分}の胸は決して大きくなくて、ほんのりと膨らんでいるぐらいなのだが、それでも柔らかく、甘い匂いがする。僕はその感触と匂いでさらに力が抜けて、彼女にぎゅっと抱き着いてしまった。,{自分}'s chest wasn't especially large; just a slight protrusion; but it was soft and had a sweet scent. This feeling and scent made feel me even weaker; and I ended up clinging to her more tightly.
```

> {自分}の胸は決して大きくなくて、ほんのりと膨らんでいるぐらいなのだが、それでも柔らかく、甘い匂いがする。僕はその感触と匂いでさらに力が抜けて、彼女にぎゅっと抱き着いてしまった。,{自分}'s chest wasn't especially large; just a slight protrusion; but it was soft and had a sweet scent. This feeling and scent made feel me even weaker; and I ended up clinging to her more tightly.

Reviewer 2: My word repetition suggestion

```suggestion
{自分}の胸は決して大きくなくて、ほんのりと膨らんでいるぐらいなのだが、それでも柔らかく、甘い匂いがする。僕はその感触と匂いでさらに力が抜けて、彼女にぎゅっと抱き着いてしまった。,{自分}'s chest wasn't especially large—little more than a bump—but it was still soft; and had a sweet smell to it. This sensation and scent made me feel even weaker; and I ended up clinging to her more tightly.
```

わたしの体に触れていたら、\nさっきの気持ちよさ、思い出してしまいましたか？,When you touched my body; did the pleasure you felt earlier come rushing back?
ぎゅって抱き着いたのも、もしかしてまたしてほしくなって、\nそれでねだるみたいに抱っこしてきたのでしょうか。\nくすくす❤甘えん坊さんですね❤,When you hugged me tightly; was that you asking me to hold you like that again? *Giggle*❤ You're quite the clingy child; aren't you?❤
あう、ううっ、それはぁ・・・❤,Ahh; uwaah; that's...❤
してほしいなら、いいですよ❤\nまたわたしの手でなでて気持ちよくしてあげます❤\n残っている分も全部出してあげますね❤,I'm willing if that's what you want.❤ I'll caress you again with my hands and make you feel good.❤ I'll make sure you release all your remaining energy; okay?❤
してほしいなら、いいですよ❤\n今度は手でなでて気持ちよくしてあげます❤\nどうしますか？残っている分も全部出しちゃいますか？,If that's what you want; then I'd be happy to.❤ This time I'll caress you with my hands and make you feel good.❤ Would you like that? Should I squeeze out every last drop?❤
やさしくやさしく甘やかすような彼女の声。それを聞いているだけでヒクヒクとペニスが震えてしまう。僕は今が試合中だということも忘れて、彼女に甘えたくて仕方がなくなってしまった。,Her voice was so gentle and sweet; drawing me in. Just hearing her made my penis tremble uncontrollably. I completely forgot about the match; all I wanted was to be spoiled by her.
うう・・・❤おねがい・・・❤\nがまんできなくて・・・❤,Hnngh... Please...❤ I can't wait any longer...❤

> まったく。頭の中がもう完全に蕩けちゃってるわね。\nいいわ、{自分}。\nその子が満足するまで気持ちよくしてあげなさい。,My; my. Your brain has gone all fuzzy; hasn't it? Alright then. {自分}; make sure to pamper him until he's totally satisfied.

Reviewer: My my is a bit of an older woman phrase I believe.
```suggestion
まったく。頭の中がもう完全に蕩けちゃってるわね。\nいいわ、{自分}。\nその子が満足するまで気持ちよくしてあげなさい。,Geez. Your brain has gone all fuzzy; hasn't it? Alright then. {自分}; make sure to pamper him until he's totally satisfied.
```

> 見られてたら恥ずかしいだろうから、\nほらバリアを周りに張ってあげる。\nこれはここまで頑張ってきた、あなたへのご褒美よ。ルーキーくん。,You'd probably feel embarrassed if people saw you like this; so I'll erect a barrier around us. This is a reward for all your hard work; little Rookie.

Reviewer: erect a barrier is weird

シルヴィアさんが何かカードを唱えると、周りが緑色のバリアに覆われ、外の様子が見えなくなった。おそらく外からも僕らの様子は見えていないのだろう。,Sylvia casts a spell using a card; and a green barrier appears around us; making it impossible to see what's happening outside. Anyone outside probably couldn't see inside; either.
僕は顔を上げて抱きついていた{自分}の顔を見る。すると彼女は目を合わせ、にこりと微笑み返してくれた。,I looked up at {自分}; whom I was hugging; and our eyes met. She smiled back down at me.
それでは、あなたがすっきりできるように、\nたっぷりご奉仕しますね。\nほら、膝の上に頭を乗せて？,Now then; I'll pamper you until you're relaxed and completely at ease. Here; rest your head on my lap. 
そうそう、言うとおりにできてえらいですね❤\nいいこ、いいこ❤\nよしよし❤,Yes; just like that. You're such a good boy; aren't you? Doing exactly as I asked.❤ Good boy; you're such a sweet child.❤ There; there~❤

> 頭をなでるだけではなく、\nおちんちんも気持ちよくしてあげますからね。\n優しく握ってしこしこしこ・・・・❤,Not just your head; I'll caress your penis too; okay? I'll place it gently in my hand and stroke...❤ 

Reviewer: Not just your head; is a weird Engrishy phrase

> 頭をなでるだけではなく、\nおちんちんも気持ちよくしてあげますからね。\n優しく握ってしこしこしこ・・・・❤,I'll caress your penis as well as your head; okay? I'll place it gently in my hand and stroke...❤ 

Reviewer 2: I think 'place' is the wrong word. Maybe take or grip

あう、ああっ、きもちいい・・・❤,Aah; aaah; it feels so good...❤

> バトルの中では無理やり搾り取るような動きで快楽攻撃を受けることが多い。しかし、彼女の手コキはそんな激しさはなく、ただただ僕を甘やかしてくれる。,When battling; I often receive pleasure attacks that are forceful and practically exploitive; yet her handjob is not nearly that intense. She simply pampers me with a gentle hand.

Reviewer: This whole line is a bit weird and could be re-worded

Author: I tried a more interpretive translation

> バトルの中では無理やり搾り取るような動きで快楽攻撃を受けることが多い。しかし、彼女の手コキはそんな激しさはなく、ただただ僕を甘やかしてくれる。,I often have to endure sexual attacks that are designed to sink their hooks in and suck the life out of me; but her handjob feels much gentler than that. It's just a gentle caressing as she pampers me.

Reviewer 2: Avoiding double 'gentle'

```suggestion
バトルの中では無理やり搾り取るような動きで快楽攻撃を受けることが多い。しかし、彼女の手コキはそんな激しさはなく、ただただ僕を甘やかしてくれる。,I often have to endure sexual attacks that are designed to sink their hooks in and suck the life out of me; but her handjob feels much softer than that. It's just a gentle caressing as she pampers me.
```

> 射精させることが目的ではなく、僕を気持ちよくすることが目的の手コキ。そこに感じる愛情のようなもののせいか、緩やかな動きなのに何倍もの快感を感じさせられてしまう。,The handjob's purpose isn't to make me cum; it's to make me feel good. Perhaps because of the affection I can feel in her movements; I feel a pleasure several times more intense despite her slow movements.

Reviewer: Same with this. Try wording it to flow better.

> 射精させることが目的ではなく、僕を気持ちよくすることが目的の手コキ。そこに感じる愛情のようなもののせいか、緩やかな動きなのに何倍もの快感を感じさせられてしまう。,She's not trying to make me cum; but simply to make me feel good. Perhaps it's because of this love and care that I sense in this handjob that; despite her slow movements; I feel an even greater sense of pleasure from it.

Reviewer 2: ```suggestion
射精させることが目的ではなく、僕を気持ちよくすることが目的の手コキ。そこに感じる愛情のようなもののせいか、緩やかな動きなのに何倍もの快感を感じさせられてしまう。,She's not trying to make me cum; but simply wants to make me feel good. Perhaps it's because of this love and care that I sense in this handjob that; despite her slow movements; I feel an even greater sense of pleasure from it.
```

> よしよし❤よしよし❤これまでずっとよく頑張りましたね❤\nおちんちん、ずっとイキたかったのに我慢して、\n一回も負けずにBランクまで戦ってえらかったですよ❤,There; there.❤ There; there.❤ You've been working hard all this time; haven't you?❤ Your penis has wanted to cum for so long; yet you've held yourself back and fought all the way up to B-rank without losing a single time. That's amazing!❤

Reviewer 2: ```suggestion
よしよし❤よしよし❤これまでずっとよく頑張りましたね❤\nおちんちん、ずっとイキたかったのに我慢して、\n一回も負けずにBランクまで戦ってえらかったですよ❤,There; there.❤ That's it. ❤ You've been working hard all this time; haven't you?❤ Your penis has wanted to cum for so long; yet you've held yourself back and fought all the way up to B-rank without losing a single time. That's amazing!❤
```

今日は負けちゃいましたけど、きっとあなたならだいじょうぶです❤\n我慢していた分、今日はたくさん気持ちよくなって、\n明日からまた頑張りましょうね❤,You may have lost this time; but I'm sure you'll do better.❤ Let's make up for it by having a really enjoyable time today; and then we'll work hard again starting tomorrow; okay?❤
うふふ❤わたしにいい子って言われてうれしいんですか？\nおちんちんがどんどん固くなってます❤\nそれにお顔もとってもうれしそう❤,Fufu~❤ Does hearing me call you a good boy make you happy? Your penis is getting harder again;❤ and your face looks so blissful; too.❤
あら、恥ずかしがらなくていいんですよ。\n今はそんな蕩けた顔をしちゃっても別にいいんです❤,Oh my; no need to feel embarrassed. It's perfectly fine to feel so entranced right now.❤
ここにはあなたとわたしだけ。\nね、何も隠す必要なんてないでしょう？,There's only you and me here. Hey; you don't need to hide anything. Okay?

> あなたが気持ちよくなれるように、\nしこしこしこ・・・❤しこしこしこ・・・❤\nそうそう、もっとその気持ちよさそうな顔を見せてください❤,I'll help you to feel really good. Stroke stroke stroke...❤ Stroke stroke stroke...❤ Yes; that's it. Keep showing me more of that blissful expression.❤

Reviewer 2: ```suggestion
あなたが気持ちよくなれるように、\nしこしこしこ・・・❤しこしこしこ・・・❤\nそうそう、もっとその気持ちよさそうな顔を見せてください❤,I'll help you feel really good. Stroke stroke stroke...❤ Stroke stroke stroke...❤ Yes; that's it. Keep showing me more of that blissful expression.❤
```

うう、きもちいい・・・❤\nクレリックさん、好き・・・❤あうう・・・❤❤,Uwaah; it feels so good...❤ Miss Cleric; I love you...❤ Hwaaah...❤❤

> あら、うれしい❤\n今日会ったばかりだけれど、\nわたしのこと好きになってくれたんですね❤,Oh my; that makes me happy!❤ We only just met today; yet you've already fallen in love with me?❤

Reviewer 2: ```suggestion
あら、うれしい❤\n今日会ったばかりだけれど、\nわたしのこと好きになってくれたんですね❤,Oh my; that makes me so happy!❤ We only just met today; and yet you've already fallen in love with me?❤
```

> 好きな子にシコシコしてもらえると、\nとっても気持ちいいでしょう？\nどんどん気持ちいいのが増えていきますね❤,Won't it feel amazing if the woman you like strokes your dick for you? The pleasure would only grow more and more; wouldn't it?❤

Reviewer 2: ```suggestion
好きな子にシコシコしてもらえると、\nとっても気持ちいいでしょう？\nどんどん気持ちいいのが増えていきますね❤,Does it feel it feel even having the woman you love stroking your penis for you? The pleasure probably only grow more and more; doesn't it?❤
```

くす❤おちんちんが、なでるたびに、\nびくんびくんって大きく動くようになりました❤\nもうすぐ出しちゃいそうなんですか？,*Giggle*❤ Your penis twitches and pulses so much every time I stroke it.❤ Are you about to cum?

> いいですよ❤マスターさん❤\nわたしに優しく癒してもらいながら、\n気持ちよく射精しましょう❤,Go ahead.❤ Sir Master;❤ let me gently comfort and soothe you; and let's enjoy our time together as you ejaculate.❤

Reviewer 2: ```suggestion
いいですよ❤マスターさん❤\nわたしに優しく癒してもらいながら、\n気持ちよく射精しましょう❤,Go ahead.❤ Dear Cardmaster~❤ Let me gently comfort and soothe you; and let's enjoy our time together as you ejaculate.❤
```

ほら、我慢しないで、体に力も入れないで…❤\nそうそう、だらーんってするの上手ですよ❤\nはい、びゅっ、びゅー❤,Come now; just relax your body; and don't hold back...❤ Yes; just like that. You're doing a wonderful job relaxing.❤ Yes; let it all out~❤

> うふふ❤すっごくたくさん出ましたね❤\nおててがどろどろになっちゃいました。\nマスターさん、もうすっきりできましたか？,Fufu~❤ You released a lot; didn't you?❤ My hands are all messy now. Do you feel better now; Sir Master?

Reviewer 2: ```suggestion
うふふ❤すっごくたくさん出ましたね❤\nおててがどろどろになっちゃいました。\nマスターさん、もうすっきりできましたか？,Fufu~❤ You released a lot; didn't you?❤ My hands are all messy now. Do you feel better now?
```

あうう、ごめんなさい・・・❤もっと・・・❤\nもっと、おねがい・・・❤,Aah; I'm sorry...❤ More...❤ One more time; please...❤
あらあら。これまで貯めてた分、\nもっともっと、びゅっ、びゅってしないと\n収まらないみたいですね❤,Oh dear~ It seems you'll need a more thorough pampering to release all that pent-up energy; won't you?❤
謝らなくていいんですよ❤\n男の子が女の子に甘えたくなっちゃうのは普通のことなんです❤,There's no need to apologize~❤ It's perfectly normal for a boy to want a woman to coddle him.❤
このまま空っぽになるまで出しちゃいましょうね❤\nほら、精液ですべりがよくなったから、\n手での奉仕ももっと気持ちいいはずです❤,Let's keep going until you've released every last drop; okay?❤ Look; your semen has made it easier to stroke; so my handjob will probably feel even better this time.❤
ああ、{自分}さん、きもちいいよぉ・・・❤❤\nすき、すきぃ・・・❤,Aah~ Miss {自分}; it feels so goooood...❤❤ I; I love youuu...❤

> こうして僕は完全に足腰が立たなくなるまで彼女に搾り取ってもらった。ひたすら甘やかされ頭をなでられながら、優しくしごき続けてくれる。闘技場で戦っていたとは思えない幸福感に包まれ、僕は意識を手放したのだった。,And so; I let her drain me completely until I was too weak to stand properly. She pampered me endlessly; caressing my head and gently stroking my dick the whole time. I eventually passed out; surrounded by this feeling of bliss I could've never imagined since fighting in this arena.

Reviewer 2: ```suggestion
こうして僕は完全に足腰が立たなくなるまで彼女に搾り取ってもらった。ひたすら甘やかされ頭をなでられながら、優しくしごき続けてくれる。闘技場で戦っていたとは思えない幸福感に包まれ、僕は意識を手放したのだった。,And so; I let her drain me completely until I was too weak to stand properly. She pampered me endlessly; caressing my head and gently stroking my dick the whole time. I eventually passed out; surrounded by this feeling of bliss I could've never imagined I'd find fighting in the arena.
```

その後、僕は医務室で目を覚ました。あとで聞いた話によると、僕が動けなくなったところで、{自分}が結界を中からノックし、結界が解除されたそうだ。そのあと闘技場のスタッフが僕をここまで運んでくれたらしい。,Later; I woke up in the infirmary. I learned that after I'd become too weak to move; someone had knocked on the barrier from the inside; and the spell was deactivated. The arena staff then apparently carried me here.

> 柔らかいベッドに寝そべりながら、僕は試合の時のことを思い出し、恥ずかしいようなむずがゆいような、言い知れない感覚に顔を赤くして、毛布に顔をうずめてしまった。,Laying on the soft mattress; I recalled the events of the match; and a strange feeling of embarrassment and cringe washed over me. Blushing; I buried my face in my blanket.

Reviewer 2: I don't think cringe is a good word choice here. I think the direct translation is unease, but guy localisation, shame might work.

あんな風に女の人に甘えたのは初めてだった。カッコよく戦わないといけないのに、昇格戦だってかかっていたのに・・・。いくら人に見えない場所だったにしてもそれでも・・・・。,That was the first time I'd ever acted so clingy towards a woman like that. I'm meant to be cool and composed when fighting; not to mention that was a promotion match... Even though it was in a private place away from prying eyes; I still shouldn't have done that...
僕は試合に負けたことを「悔しい」と思わなければいけないはずだが、頭の中は痺れたように心地よく、悔しさも後悔もなかなか湧いてこなかった。こんな調子でBランクを戦っていけるのだろうか。,I should've felt frustrated about losing the match; but my mind felt pleasantly numb; and I couldn't really muster up any frustration nor regret. Could I really compete in the B-rank matches in this state?
それからしばらくして,Some time later
はぁ、また来たの？{主人公}。\nまったく、あなた、{自分}にはまりすぎよ。,*Sigh*; you're here again; {主人公}? Honestly; you're getting way too addicted to {自分}.

> 女の子にあまり優しくされたことがなかったのかしら。\nまああなたの精液は良質だから、別にいいといえばいいんだけど、\n無敗のルーキーがこんな風になっちゃったのは少し責任を感じるわ。,Were you treated badly by girls your whole life or something? Well; your semen is high-quality; so I guess it's fine. Still; I can't help but feel a bit guilty for turning the Undefeated Rookie into this.

Reviewer: ```suggestion
女の子にあまり優しくされたことがなかったのかしら。\nまああなたの精液は良質だから、別にいいといえばいいんだけど、\n無敗のルーキーがこんな風になっちゃったのは少し責任を感じるわ。,Were you treated badly by girls your whole life or something? Well; your semen is high-quality; so I guess it's fine. Still; I can't help but feel a bit guilty for turning the Undefeated Rookie into this kind of person.
```

> 女の子にあまり優しくされたことがなかったのかしら。\nまああなたの精液は良質だから、別にいいといえばいいんだけど、\n無敗のルーキーがこんな風になっちゃったのは少し責任を感じるわ。,Were you treated badly by girls your whole life or something? Well; your semen is high-quality; so I guess it's fine. Still; I can't help but feel a bit guilty for turning the Undefeated Rookie into this kind of person.

Reviewer 2: ```suggestion
女の子にあまり優しくされたことがなかったのかしら。\nまああなたの精液は良質だから、別にいいといえばいいんだけど、\n無敗のルーキーがこんな風になっちゃったのは少し責任を感じるわ。,Were you treated that badly by girls your whole life or something? Well; your semen is high-quality; so I guess it's fine. Still; I can't help but feel a bit responsible for how far the Undefeated Rookie has fallen. 
```

> あのときご褒美だなんて言って、\n甘えさせなければかったのかしら。\n人を育てるのって難しいのね。,Maybe I shouldn't have spoiled you by calling it a reward that time. Child-rearing sure is difficult.

Reviewer: 人を育てる is more like to bring up a person
```suggestion
あのときご褒美だなんて言って、\n甘えさせなければかったのかしら。\n人を育てるのって難しいのね。,Maybe I shouldn't have spoiled you by calling it a reward that time. Bringing up people right sure is difficult.
```

> あのときご褒美だなんて言って、\n甘えさせなければかったのかしら。\n人を育てるのって難しいのね。,Maybe I shouldn't have spoiled you by calling it a reward that time. Child-rearing sure is difficult.

Reviewer 2: ```suggestion
あのときご褒美だなんて言って、\n甘えさせなければかったのかしら。\n人を育てるのって難しいのね。,Maybe I shouldn't have spoiled you by calling it a reward that time. Caring for another person sure is difficult.
```

はいはい、わかったわかった。\nあの子がいないと辛くて泣きそうなんでしょう？\nほら、{自分}を召喚。,Yes yes; I know I know. You're feeling lonely and on the verge of tears without her around; right? Here; I summon {自分}.
また会いに来てくれたんですね。{主人公}くん❤\n今日も手でしてほしいんですか？,You're here to see me again; dear {主人公}?❤ Do you want me to do it with my hands again today?
うふふ、それじゃあ、向こうの部屋に行きましょう？\nマスターがお勉強している間に、\nわたしたちはマスターに魔力供給です❤,Fufu; well then; shall we head to the other room? While Master is doing research; we'll be supplying her with magical energy.❤

> 二人きりのお部屋で、しこしこ、しこしこ…❤しましょうね❤\nくす、もう我慢できないって顔してます❤,With just the two of us in the room... Are you ready?❤ We'll be stroking it; and stroking it...❤ *Giggle*; you look like you're at your limit already.❤

Reviewer: We'll be stroking it; and stroking it...❤ is a bit weird

> 二人きりのお部屋で、しこしこ、しこしこ…❤しましょうね❤\nくす、もう我慢できないって顔してます❤,With just the two of us in the room... Are you ready?❤ I'll keep stroking that part of you; over and over...❤ *Giggle*; you look like you're at your limit already.❤

Reviewer 2: Small change to make sure two sentences in a row didn't end with 'ready'

```suggestion
二人きりのお部屋で、しこしこ、しこしこ…❤しましょうね❤\nくす、もう我慢できないって顔してます❤,With just the two of us in the room... Are you ready?❤ I'll keep stroking that part of you; over and over...❤ *Giggle* You look like you're already at your limit~❤
```

{主人公}くんは確かにあまり試合で勝てなくなってしまいましたけど\nその分、人に甘えることを覚えたんです❤\n暴力に明け暮れるよりとても素敵なことじゃないですか❤,My sweet {主人公} may not have won many matches lately; but in return; you've learned to rely on others more.❤ Isn't that much better than spending all your time engaging in violence?❤
だから、シルヴィア様は後悔する必要なんてありませんよ。\n{主人公}くんは正しく導かれています❤,Therefore; Madam Sylvia doesn't need to feel bad for anything. Little {主人公} has been guided on the right path thanks to you.❤

> それじゃ、はい。わたしの手を握ってください。\nお部屋まで一緒に歩いていきましょうね❤,Well then; here. Take my hand. Let's walk to our room together; okay?❤

Reviewer 2: What do you think of 'now then' instead of 'well then'

それからというもの、頻繁にシルヴィアさんのところに通っては彼女を召喚してもらい、二人きりの時間を過ごしている。魔力をたくさん吸われてしまうから修行はもちろんほとんどできていない。,From then on; I'd frequently visit Miss Sylvia and have her summon {自分} so I can spend some alone time with her. Since so much of my energy gets drained by her; I haven't been able to do much training at all.
ランクも日に日に落ちて行って、もうすぐＣランクからも転落しそうな勢いだ。しかし、そんな状況になっても{自分}は優しく甘やかしてくれる。僕はますます彼女から離れられなくなっていった。,My rank dropped day after day; until it looked like I'd fall out of C-rank altogether. But even then; {自分} continued to be kind and indulgent to me. I became even more dependent on her.
こうして無敗でＢランクにまで駆け上がった僕の活躍は終わり、優しい愛に包まれながらその幕を閉じたのだった。,And so; my journey of rising all the way to B-rank without a single loss came to an end; concluding with a gentle embrace.
えっと、気持ちよくしてほしい、ですか？\nあらあら、もう戦いに疲れてしまったんですね。\nいいですよ❤その疲れた体、わたしが癒してあげます❤,Um; you want me to make you feel good; you said? Oh dear; it seems you've been exhausted from all this fighting; haven't you? Alright; then.❤ I'll help soothe your weary soul.❤
あら？うずくまってどうされたのですか？\n暴力的な戦闘はしていないはずですが、\nどこか体をぶつけてしまったとか・・・。,Oh? Why are you bending over like that? I don't think I struck you particularly violently; did you perhaps bump into something...?

> そうそう。しこしこ上手ですね❤\nあなたが気持ちよくなれるように、\n頭を後ろからなでてあげますからね❤,Yes yes; you're stroking so well; aren't you!❤ Let me comfort you by caressing the back of your head.❤

Reviewer 2: ```suggestion
そうそう。しこしこ上手ですね❤\nあなたが気持ちよくなれるように、\n頭を後ろからなでてあげますからね❤,Yes; that's it. You're stroking so well~❤ Let me comfort you by caressing the back of your head.❤
```

うふふ❤たくさん出せましたね❤\n戦いを捨て、快感に身を増せることができて、\nとてもいい子ですよ❤えらい❤えらい❤,Fufu~❤ You let out a lot; didn't you?❤ Giving up the fight and accepting pleasure; you're such a good child!❤ You did so well.❤ Amazing.❤

> あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n幸せで素晴らしい結果ですね❤ふふ❤,I've recovered too; thanks to your energy. You were healed through pleasure; and I was healed with your energy. It's an outcome that leaves everyone happy; isn't it~❤

Reviewer 2: ```suggestion
あなたのエナジーをいただいて、わたしも回復しました。\n快感であなたが癒されて、エナジーでわたしが癒される、\n幸せで素晴らしい結果ですね❤ふふ❤,I've recovered too; thanks to your energy. You were healed through pleasure; and I was healed with your energy. It's an outcome that leaves everyone happy; don't you think?❤
```

> 自分で負けてしまいましたが気に病むことはありません。\n男の子が女性に甘えるのは、当然のことですから❤,You may have lost all on your own; but that's okay. It's only natural for a boy to be affectionate to a woman.❤

Reviewer: 男の子が女性に甘える -> boys to be pampered by women
```suggestion
自分で負けてしまいましたが気に病むことはありません。\n男の子が女性に甘えるのは、当然のことですから❤,You may have lost all on your own; but that's okay. It's only natural for boys to be pampered by women.❤
```
